jinja2/react_base.html

```diff
@@ -152,7 +152,8 @@
     <!--[if lte IE 8]>{% javascript 'selectivizr' %}<![endif]-->
 
     {{ providers_media_js() }}
-    <script src="{{ react_i18n(request.LANGUAGE_CODE) }}"></script>
+    <!-- strings used by javascript modules. Hopefully we can remove soon -->
```

Comment:
```suggestion
    {# strings used by javascript modules. Hopefully we can remove soon #}
```

Jinja comment instead since this is a comment that doesn't need to be seen beyond our code base.